README.md

```diff
@@ -7,155 +7,30 @@
 ```
 
 
-Use the same terminal for everything. The main reason for this plugin is to
```

Comment:
Please don't add unnecessary changes.

Reply:
Ok, never did this before. ;)
I try to roll it back.